README.md

```diff
@@ -25,17 +25,22 @@ jobs:
         uses: bump-sh/github-action@0.2
         with:
           id: <BUMP_DOC_ID>
-          token: <BUMP_DOC_TOKEN>
+          token: ${{secrets.BUMP_TOKEN}}
```

Comment:
what do you think about keeping the example using the following `<BUMP_DOC_TOKEN>`?